ferrocene/doc/internal-procedures/src/upstream-pulls.rst

```diff
@@ -0,0 +1,82 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Upstream Pulls
+==============
+
+What is an Upstream Pull
+------------------------
+
+An "upstream pull" is a pull request that pulls in commits from the upstream Rust
+repository back into the Ferrocene repository to keep them in sync. This
+procedure is usually performed by a periodic Github Action that opens a pull
+request automatically. Sometimes merge conflicts may occur in which case the conflict
+needs to to be resolved manually.
+
+Upstream Pull without merge conflicts
+-------------------------------------
+
+To sign off a automatic upstream pull without merge conflicts, the reviewer only
+needs to approve the PR and comment `bors merge`. This will trigger the "full" CI
+workflow. If the workflow succeeds, the Upstream Pull is getting merged.
+
+Upstream Pull with merge conflicts
+----------------------------------
+
+If an automatic upstream pull creates merge conflicts, the automation commits the
+merge conflicts and they need to be resolved manually.
+
+Checkout PR
+^^^^^^^^^^^
+
+Start by checking out the PR. The name of the branch is
+`automation/pull-upstream/<ID>`, e.g. `automation/pull-upstream/dfvjj2s4`::
+
+    git fetch
+    git switch automation/pull-upstream/<ID>
+
+Fix conflict
+^^^^^^^^^^^^
+
+To find the conflicting files run following command::
+
+  python ferrocene/ci/scripts/detect-conflict-markers.py
```

Comment:
```suggestion
  ferrocene/ci/scripts/detect-conflict-markers.py
```

Nit: the path to the Python executable is not needed.

Reply:
Oh what?!?!?? 🤯